src/test/java/com/proptechos/utils/DataLoader.java

```diff
@@ -0,0 +1,27 @@
+package com.proptechos.utils;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.stream.Collectors;
+
+public class DataLoader {
+
+  public static String loadPagedDevices() {
+    String dataFile = "data/device/paged.json";
+    InputStream is = DataLoader.class.getClassLoader().getResourceAsStream(dataFile);
+    return new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8))
+        .lines()
+        .collect(Collectors.joining("\n"));
+  }
+
+  public static String loadSingleDevice() {
+    String dataFile = "data/device/single.json";
+    InputStream is = DataLoader.class.getClassLoader().getResourceAsStream(dataFile);
+    return new BufferedReader(new InputStreamReader(is, StandardCharsets.UTF_8))
+        .lines()
+        .collect(Collectors.joining("\n"));
+  }
```

Comment:
Please consider template method, with one abstract method getDataFile, and two implementations > SingleDeviceDataLoader and PagedDeviceDataLoader

Reply:
but this is test. Maybe it is not worth to do such changes. Never mind